packages/flutter_goldens_client/lib/skia_client.dart

```diff
@@ -682,9 +683,6 @@ class SkiaGoldDigest {
 
   /// Create a digest from requested JSON.
   factory SkiaGoldDigest.fromJson(Map<String, dynamic> json) {
-    if (json == null)
-      return null;
-
```

Comment:
@Piinks: does this particular change looks good to you? I haven't found place where this constructor is call with null. 

In NNBD a factory constructor can not return `null`. If we want to return `null` the only solution is to replace this factory constructor by a static method.

Reply:
I think this will be ok. The only use of this factory is above on line 569. :)